includes/wc-core-functions.php

```diff
@@ -157,7 +157,12 @@ function wc_update_order( $args ) {
  * @param string $name Template name (default: '').
  */
 function wc_get_template_part( $slug, $name = '' ) {
-	$cache_key = sanitize_key( implode( '-', array( 'template-part', $slug, $name, WC_VERSION ) ) );
+	if ( isset( $_SERVER['SERVER_NAME'] ) ) {
+		$uniquepath = sanitize_key( $_SERVER['SERVER_NAME'] );
```

Comment:
This variable name don't follow WP standards, should use `_` like `$unique_path`. And this should not help everyone, probably better have an filter here too.

```
$server_name = isset( $_SERVER['SERVER_NAME'] ) ? sanitize_key( $_SERVER['SERVER_NAME'] ) : '';

$cache_key = sanitize_key( apply_filters( 'woocommerce_get_template_part_cache_key', implode( '-', array( 'template-part', $slug, $name, WC_VERSION, $server_name ) ), $slug, $name, WC_VERSION, $server_name ) );
```

Reply:
Thanks for the feedback @claudiosanches , updated my PR here https://github.com/woocommerce/woocommerce/pull/23783/commits/217a95e937b6a2b5e54b0e5ee1023c4ae6cc7946